frontend/src/routes/Waiting.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTranslation } from "react-i18next";
+
+import { Card } from "../ui/Card";
+import { FiTruck } from "react-icons/fi";
+import { keyframes } from "@emotion/react";
+
+export const WaitingPage: React.FC<{ type: "video" | "series" }> = ({ type }) => {
```

Comment:
I don't think this should be in `src/routes`. In routes, there are only... well, actual routes. This should go in `src/ui`.

Reply:
I got confused by `NotFound`. But of course that's a route as well.